### PR TITLE
add haproxy to rosdep/base.yaml

### PR DESCRIPTION
**Pull request by mqcmd196**
I want to add haproxy to base.yaml.

Links:
- https://packages.debian.org/buster/haproxy (debian)
- https://src.fedoraproject.org/rpms/haproxy (fedora)
- https://packages.ubuntu.com/bionic/haproxy (ubuntu)

### CODE REVIEW

**Review by tfoote — APPROVED** [review]
Thanks for the links. 

This looks good. Before the second review if you have time to add `net-proxy/haproxy` for gentoo that would be great: https://packages.gentoo.org/packages/net-proxy/haproxy

**Comment by mqcmd196** [comment]
Thanks!

File: rosdep/base.yaml [hunk 1]
```diff
@@ -1664,6 +1664,10 @@ gv:
   gentoo: [app-text/gv]
   nixos: [gv]
   ubuntu: [gv]
+haproxy:
+  debian: [haproxy]
+  fedora: [haproxy]
+  ubuntu: [haproxy]
```
Comment (mabelzhang) [inline rosdep/base.yaml:1670]:
```suggestion
  ubuntu: [haproxy]
  gentoo: [net-proxy/haproxy]
```

**Review by mabelzhang — COMMENTED** [review]
As above, please add gentoo. Otherwise looks good.

**Comment by mqcmd196** [comment]
I updated.

**Review by mabelzhang — APPROVED** [review]
Merge on green CI

**Pull request merged**